tools/protoxform/protoprint.py

```diff
@@ -82,8 +82,9 @@ def clang_format(contents):
     Returns:
         clang-formatted string
     """
+    clang_format_path = os.getenv("CLANG_FORMAT", "clang-format-11")
```

Comment:
The default will now change from `clang-format` to `clang-format-11` for consistency with `https://github.com/envoyproxy/envoy/blob/main/tools/code_format/check_format.py#L160`.